<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QDialog" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SignUp</string>
  </property>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>456</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;请输入注册信息&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="signupPushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>520</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="goToLoginPushButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>520</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>返回登录</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>337</width>
     <height>299</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>用户名:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>所在学院:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Student ID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>性别:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>密码:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>再次输入密码:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>身份类型:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>出生日期:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="userLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="depLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="idLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="genderLineEdit">
        <property name="placeholderText">
         <string>boy/girl</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pwd1LineEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pwd2LineEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="stuRadioButton">
          <property name="text">
           <string>Student</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="teaRadioButton">
          <property name="text">
           <string>Teacher</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="birLineEdit">
        <property name="placeholderText">
         <string>2000/01/00</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
